styles/Home.module.scss

```diff
@@ -32,3 +34,111 @@
 .title {
   text-align: center;
 }
+
+.gridContainer1 {
```

Comment:
gridContainer found, but no gridContainer1 and gridContainer2. Do you name it differently?